client/reader/recent/style.scss

```diff
@@ -203,10 +203,10 @@
 			padding-bottom: 132px;
 		}
 
-		.reader-full-post__sidebar,
-		.reader-full-post__author-block,
```

Comment:
These class styles shouldn't be necessary because we're removing these elements from the component based on the `layout` prop.